handler/openid/flow_hybrid.go

```diff
@@ -54,7 +54,9 @@ func (c *OpenIDConnectHybridHandler) HandleAuthorizeEndpointRequest(ctx context.
 	if !(ar.GetResponseTypes().Matches("token", "id_token", "code") || ar.GetResponseTypes().Matches("token", "code") || ar.GetResponseTypes().Matches("id_token", "code")) {
 		return nil
 	}
-
+	if ar.GetResponseMode() == fosite.ResponseModeNone {
```

Comment:
I think this should not run if `fosite.ResponseModeQuery` and maybe also not with the `FormPost` one. Please confirm @mitar . Also use `ar.SetDefaultResponseMode`.

Reply:
added `ar.SetDefaultResponseMode`. Using `query` in this flow is already restricted but do we want to restrict `form_post`? According to the [spec](https://openid.net/specs/oauth-v2-form-post-response-mode-1_0.html#Security), it seems not necessary to restrict `form_post` as it more secure than `query` and  `fragment`.